app/javascript/enableSubmitOnInput.js

```diff
@@ -0,0 +1,22 @@
+const enableSubmitOnInput = () => {
+  const form = document.querySelector(".ai-annotation-form")
+  if (!form) return
+
+  const textInput = document.querySelector(".text-input")
+  const promptInput = document.querySelector(".prompt-input")
+  const submitButton = document.querySelector(".submit-button")
+
+  const toggleSubmitButton = () => {
+    if (form.checkValidity()) {
+      submitButton.disabled = false
+    } else {
+      submitButton.disabled = true
+    }
+  }
+
+  textInput.addEventListener("input", () => toggleSubmitButton())
+  promptInput.addEventListener("input", () => toggleSubmitButton())
```

Comment:
引数を渡さない場合は、関数名だけ渡せば十分です。

```suggestion
  promptInput.addEventListener("input", toggleSubmitButton)
```

Reply:
修正しました。
https://github.com/yush-nh/textae-ai/pull/12/commits/e4cd62ef526ef5f811320a2c0ccdcfb6cec7ba82